docs/software_setup/sd_card.md

```diff
@@ -1,10 +1,20 @@
 This page details the software setup required to run JetBot.
 
+???+ info
```

Comment:
Remove this admonition, the disclaimer doesn't seem necessary given we list these steps as associated with the DIY jetbot in getting started guide